core/meta/inc/TClass.h

```diff
@@ -173,7 +176,8 @@ friend class TProtoClass;
    TListOfDataMembers *fData;            //linked list for data members
 
    std::atomic<TListOfEnums*> fEnums;        //linked list for the enums
-   TListOfFunctionTemplates  *fFuncTemplate; //linked list for function templates [Not public until implemented as active list]
+   TListOfFunctionTemplates  *fFuncTemplate; //linked list for function templates
```

Comment:
Oops, sorry, I will revert this part then. Is it ok to just put the bracketed text on the next line, or does it need to be all in one line?